src/de/jost_net/JVerein/server/LastschriftImpl.java

```diff
@@ -237,7 +238,24 @@ public void setOrt(String ort) throws RemoteException
   @Override
   public String getStaat() throws RemoteException
   {
-    return (String) getAttribute("staat");
+    String code = getStaatCode();
+    if (Staat.getByKey(code) != null)
+    {
+      return Staat.getByKey(code).getText();
+    }
+    // Wenn der Code nicht vorhanden ist, nehmen wir
+    // zur Kompatibilit�t den Text wie er in der DB Steht
+    return code;
+  }
+
+  @Override
+  public String getStaatCode() throws RemoteException
+  {
+    String code = (String) getAttribute("staat");
+    // Wenn noch das ganze Land drin steht, bestimmen wir den Code
+    if (Staat.getByText(code) != null)
+      return Staat.getByText(code).getKey();
+    return code;
   }
```

Comment:
Ich fände es wirklich super, wenn wir code duplication vermeiden (hier fünfmal) und diese beiden Methoden als [default methods](https://www.javatpoint.com/default-and-static-methods-in-interface-java-8) in ein Interface auslagern würden.
Das Attribut ließe sich dann auch mit einer overload method angeben und auch für "ktoistaat" nutzen.